source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+Incident Response Application (EE, Closed Alpha)
```

Comment:
I'm open to feedback on the length of this page and whether we'd want to break into separate smaller pages. 

As a note, 80% of this is configuration, as I wanted to make it all-inclusive for the JSON configuration with all info available in one place. The configuration will be much simpler once it’s moved to the UI.